NumberValidator/Validators/IN/vid.cs

```diff
@@ -0,0 +1,149 @@
+﻿using System;
+using System.Linq;
+using System.Text.RegularExpressions;
+
+namespace VidHandling
+{
+    public class InvalidLengthException : Exception { }
+    public class InvalidFormatException : Exception { }
+    public class InvalidChecksumException : Exception { }
+
+    public static class Vid
+    {
+        private static readonly Regex VidRegex = new Regex(@"^[2-9][0-9]{15}$");
+
+        public static string Compact(string number) =>
```

Comment:
why new method implementation? what functionality is not provided by `Clear` method?

Reply:
its corrected